system-configuration/src/dynamic_store.rs

```diff
@@ -193,6 +194,19 @@ impl SCDynamicStore {
         }
     }
 
+    /// Returns the key-value pairs that represent the current internet proxy settings. Or `None` if
+    /// no proxy settings have been defined or if an error occured.
+    pub fn get_proxies(&self) -> Option<CFDictionary<CFString, CFType>> {
```

Comment:
Not sure if you guys specify the documentation URLs in here, but the keys returned in that dict can be found at https://developer.apple.com/documentation/systemconfiguration/1517088-scdynamicstorecopyproxies

Reply:
The other functions didn't have any more docs specified than this either, so that's the style I used. In addition to that, there are more keys that can be in this dictionary than the linked documentation lists: all constants starting with `kSCPropNetProxies` can be keys here. As those constants are already defined as `system-configuration-sys::schema_definitions::kSCPropNetProxies*`, I didn't define those again in this PR.